fuel-indexer/indexer/src/api.rs

```diff
@@ -0,0 +1,127 @@
+use crate::{IndexerConfig, SchemaManager};
+use async_std::sync::{Arc, RwLock};
+use fuel_indexer_schema::db::{
+    graphql::{GraphqlError, GraphqlQueryBuilder},
+    tables::Schema,
+};
+use serde::Deserialize;
+use thiserror::Error;
+use tokio_postgres::{connect, types::Type, NoTls};
+use tracing::error;
+use warp::{http::StatusCode, reply::with_status, Filter, Reply};
```

Comment:
Ideally, we could use the same server stack across both fuel-core and fuel-indexer. We're currently using axum in other places.

Reply:
This would make it really easy to combine API handlers from both projects into a single webserver as well.